libretro.cpp

```diff
@@ -1715,12 +1732,13 @@ void retro_set_environment(retro_environment_t cb)
 
    static const struct retro_variable vars[] = {
       { "pce_fast_cdimagecache", "CD Image Cache (Restart); disabled|enabled" },
-      { "pce_nospritelimit", "No Sprite Limit; disabled|enabled" },
+      { "pce_nospritelimit", "No Sprite Limit (Restart); disabled|enabled" },
+      { "pce_hoverscan", "Horizontal Overscan (Restart); disabled|enabled" },
```

Comment:
Yeah I don't know what can be done here. 
I just tested those settings and they work only if after you changed them you do "close" in QuickMenu, then start the game/core again.
That's why the "restart" part is written to hint that fact.